users/vieting/experiments/librispeech/librispeech_100_ctc/fairseq_finetuning/config_01_finetune.py

```diff
@@ -0,0 +1,185 @@
+"""
+Config for finetune experiments on LibriSpeech using wav2vec 2.0.
+"""
+import os.path
+
+from sisyphus import tk, gs
+import recipe.i6_core.datasets.librispeech as librispeech
+from recipe.i6_core.audio.encoding import BlissChangeEncodingJob
+from recipe.i6_core.tools.git import CloneGitRepositoryJob
+from recipe.i6_experiments.users.engler.fairseq.training import FairseqHydraConfig, FairseqHydraTrainingJob
+from recipe.i6_experiments.users.vieting.jobs.fairseq import CreateFairseqLabeledDataJob
+from recipe.i6_experiments.users.vieting.experiments.librispeech.librispeech_960_pretraining.wav2vec2.fairseq import SetupFairseqJob
+
+python_path = tk.Path("/work/asr3/vieting/hiwis/pletschko/miniconda3/envs/fairseq_python38/bin/python")
```

Comment:
This is what you still plan to fix, right? We might run the setup with an apptainer image and then just use `/usr/bin/python3`.